ci-script-windows.sh

```diff
@@ -56,13 +56,8 @@ execute(){
 build_darktable() {
     cd "$(dirname "$0")"
 
-    # for RawSpeed submodule
-    git submodule init
-    git submodule update
-
-    mkdir artifacts
     mkdir build && cd build
-    cmake -G "MSYS Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/c/projects/darktable/artifacts ../.
+    cmake -G "MSYS Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/c/projects/darktable/build ../.
```

Comment:
Right, i remember disscussing that.
And it does not happen with usual make?
IIRC the agreement was that we need to prepend `bash` before each script we call from cmake
I.e. `#!/bin/bash` does not work because the script is not being handed to the proper shell in the first place.